src/local-storage.js

```diff
@@ -0,0 +1,64 @@
+import { editTodo, removeTodo } from './edit-remove.js';
+import { myArray } from './index';// eslint-disable-line
+
+const todoMainContainer = document.querySelector('.todos-container');
+
+// get from localstorage function
+const getFromLocal = () => {
+  const data = JSON.parse(localStorage.getItem('list'));
+  data.map(i => {// eslint-disable-line
```

Comment:
- [ ] Please kindly do not disable linters, fix linters errors helps programmers to write clean and clear codebase 😁 😉